english/khudi/author.md

```diff
@@ -0,0 +1,2 @@
+It was written by Allama Iqbal.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.